src/samplers/tcp/mod.rs

```diff
@@ -165,6 +165,14 @@ impl Tcp {
                     .handler("trace_tcp_drop")
                     .function("tcp_drop")
                     .attach(&mut bpf)?;
+                bcc::Kprobe::new()
+                    .handler("trace_tlp")
+                    .function("tcp_send_loss_probe")
+                    .attach(&mut bpf)?;
+                bcc::Kprobe::new()
+                    .handler("trace_rto")
+                    .function("tcp_retransmit_skb")
```

Comment:
I think the right probe here should be 
tcp_retransmit_timer
This function is called when a retransmit time expires, and is when we should count a RTO.
https://elixir.bootlin.com/linux/v5.4/source/net/ipv4/tcp_timer.c#L431

Reply:
Thanks for noticing this. I ran some more tests and you're right, `tcp_retransmit_timer` is the correct probe.